/home/athira/.local/lib/python3.10/site-packages/setuptools/_distutils/dist.py:289: UserWarning: Unknown distribution option: 'tests_require'
  warnings.warn(msg)
[2;36m[08/26/25 15:54:10][0m[2;36m [0m[1;31mERROR   [0m listing git files failed - pretending     ]8;id=288395;file:///home/athira/.local/lib/python3.10/site-packages/setuptools_scm/_file_finders/git.py\[2mgit.py[0m]8;;\[2m:[0m]8;id=379098;file:///home/athira/.local/lib/python3.10/site-packages/setuptools_scm/_file_finders/git.py#26\[2m26[0m]8;;\
[2;36m                    [0m         there aren't any                          [2m         [0m
